### PR TITLE
Make distro compile

### PR DESCRIPTION
**Pull request by feclare**

### CODE REVIEW

File: cmd/distro/main.go [hunk 1]
```diff
@@ -39,20 +39,20 @@ func main() {
...
-	client.InitMongoRepository(repo)
+	distro.InitMongoRepository(repo)
 
 	errs := make(chan error, 2)
 
 	go func() {
 		p := fmt.Sprintf(":%d", cfg.Port)
-		logger.Info("Staring Export Client", zap.String("url", p))
-		errs <- http.ListenAndServe(p, client.HTTPServer())
+		logger.Info("Staring Export Distro", zap.String("url", p))
```
Comment (drasko) [inline cmd/distro/main.go]: Fix typo - "Starting". While you are there, you can fix the same type in `client`, if exists.

File: cmd/distro/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+distro
```
Comment (drasko) [inline cmd/distro/.gitignore]: Do not add .gitignore in different directories. Rather add binary to the top .gitignore file and drop this file.

File: cmd/distro/main.go (hunk 1 above)
Comment (feclare) [reply cmd/distro/main.go]: Fixed both commands

File: cmd/distro/.gitignore (hunk 2 above)
Comment (feclare) [reply cmd/distro/.gitignore]: Moved to the top gitignore and added commands.

File: .gitignore [hunk 3]
```diff
@@ -22,3 +22,5 @@ tags
 
 # Binary
 mainflux-http-sender
+cmd/client/client
+cmd/distro/distro
```
Comment (drasko) [inline .gitignore]: End files with the newline (https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

Comment (feclare) [reply .gitignore]: Amended the commit to add the newline

**Comment by drasko** [comment]
LGTM

**Comment by drasko** [comment]
Merged. Thanks @feclare!

**Pull request merged**